samples/azure-quantum/resource-estimation/hello-re-qiskit.ipynb

```diff
@@ -0,0 +1,600 @@
+{
```

Comment:
Similar comment as elsewhere in PR, can we name these similarly to their final name in gallery? So if not using hello world in sample title, we may want to rename file to just be re-qiskit.ipynb